src/ai/backend/agent/stats.py

```diff
@@ -366,6 +366,7 @@ async def collect_container_stat(
                     cid = info['container_id']
                 except KeyError:
                     log.warning('collect_container_stat(): no container for kernel {}', kid)
+                    continue
```

Comment:
How about this for better readability?
```suggestion
                else:
```